packages/bpk-docs/src/pages/NativeSwitchPage/NativeSwitchPage.js

```diff
@@ -30,22 +30,20 @@ const components = [
   {
     id: 'default',
     title: 'Default',
-    screenshots: [
+    videos: [
       {
         title: 'iOS',
         width: 750,
-        height: 1334,
-        src: `/${iosScreenshotDefault}`,
-        altText: 'iOS Default Switch Component',
+        height: '100%',
```

Comment:
I just commented out the height part and it still works for me on Chrome. I reckon it's ignoring `100%` because that's not a valid value for height (see my above comment). So maybe we can remove this.

Reply:
I'll remove it see what happens, cheers!

Reply:
![perfection](https://media.giphy.com/media/9WXyFIDv2PyBq/giphy.gif)